### ISSUE TITLE
Cannot merge two updates into one

### ISSUE THREAD

**Issue by voxik** [fedora-infra/bodhi#842]
@junaruga created two updates [1, 2]. But they should be actually one, since rubygem-typhoeus depends on rubygem-ethon. So we unpushed the rubygem-ethon, trying to add it as another package into already existing rubygem-typhoeus update. Unfortunately, this fails with "Builds : Update for rubygem-ethon-0.9.0-1.fc24 already exists" message. This looks to be wrong.

[1] https://bodhi.fedoraproject.org/updates/FEDORA-2016-20a3699002
[2] https://bodhi.fedoraproject.org/updates/FEDORA-2016-f148a294f9

**Comment by lmacken** [comment]
Yeah, this is a known limitation right now. Bodhi is currently unable to remove a build from an update which would leave it empty. I've seen multiple maintainers hit this issue in the past few weeks, so it's definitely something we should figure out.

**Comment by hroncok** [comment]
Can this or https://github.com/fedora-infra/bodhi/issues/1809 be prioritized please? Handling subsequent updates that need to be manually synchronized is a huge PITA:

https://bodhi.fedoraproject.org/updates/FEDORA-2020-5d6c8b0c64

**Comment by cverna** [comment]
Same as #1809 :(
